specification/purview/Azure.Analytics.Purview.DataMap/tspconfig.yaml

```diff
@@ -0,0 +1,36 @@
+parameters:  
+  "service-dir":
+    default: "sdk/azure.analytics.purview.datamap"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/Azure.Analytics.Purview.DataMap/{version-status}/{version}/openapi.json"
+  "@azure-tools/typespec-python":    
+    package-dir: "azure-analytics-purview-datamap"
+    package-name: "{package-dir}"  
+    package-mode: dataplane
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Analytics.Purview.DataMap"
+    namespace: "{package-dir}"
+    clear-output-folder: true
+    model-namespace: false
+  "@azure-tools/typespec-ts":
+    package-dir: "azure-analytics-purview-datamap-rest"
+    generateMetadata: true
+    generateTest: true
+    packageDetails:
+      name: "@azure-rest/azure-analytics-purview-datamap-rest"
```

Comment:
Usually the package name would be `@azure-rest/{service-name}-{module}` e.g: @azure-rest/analytics-purview-datamap

Reply:
Thanks. Updated